Windows build fails: Errors in `morpion_solitaire.cc`

I'm trying to build OpenSpiel on windows.

I followed the instructions [in the docs](https://github.com/deepmind/open_spiel/blob/master/docs/windows.md), cloning 5eaf401f7d08d68285fe214ab5cf5a741807ea6e.

CMake runs properly, but I get 8 syntax errors, all from the file `morpion_solitaire.cc`. It seems this file uses some fancy C++ notation I (or VS at least) am not familiar with - using `...` in a `case` block. My complete build log is too long for a GitHub issue comment, but I've paste-binned it here: https://pastebin.com/swekt5XZ.

The relevant build log snippets are;

```C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(67): error C2143: syntax error: missing ':' before '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(67): error C2059: syntax error: '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(69): warning C4244: 'argument': conversion from 'open_spiel::Action' to 'int', possible loss of data
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(70): warning C4244: 'argument': conversion from 'open_spiel::Action' to 'int', possible loss of data
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(73): error C2143: syntax error: missing ':' before '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(73): error C2059: syntax error: '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(80): error C2143: syntax error: missing ':' before '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(80): error C2059: syntax error: '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(87): error C2143: syntax error: missing ':' before '...'
C:\Development\open_spiel\open_spiel\games\morpion_solitaire.cc(87): error C2059: syntax error: '...'
```

My VS info is as follows;

```Microsoft Visual Studio Community 2019
Version 16.11.9
VisualStudio.16.Release/16.11.9+32106.194
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

Visual C++ 2019   00435-60000-00000-AA301
Microsoft Visual C++ 2019

ASP.NET and Web Tools 2019   16.11.94.52318
ASP.NET and Web Tools 2019

C# Tools   3.11.0-4.21602.3+fc14a1355c0461af5110b74b26f6478f22d26565
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

IntelliCode Extension   1.0
IntelliCode Visual Studio Extension Detailed Info

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual C++ Wizards   1.0
Microsoft Visual C++ Wizards

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

NuGet Package Manager   5.11.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

Test Adapter for Boost.Test   1.0
Enables Visual Studio's testing tools with unit tests written for Boost.Test.  The use terms and Third Party Notices are available in the extension installation directory.

Test Adapter for Google Test   1.0
Enables Visual Studio's testing tools with unit tests written for Google Test.  The use terms and Third Party Notices are available in the extension installation directory.

TypeScript Tools   16.0.30526.2002
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   3.11.0-4.21602.3+fc14a1355c0461af5110b74b26f6478f22d26565
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake
```

--- a/docs/windows.md
+++ b/docs/windows.md
@@ -59,7 +59,8 @@ rest) from any location, you will need to add to your PYTHONPATH the root
 directory and the `open_spiel` directory. Open
 [Windows environment variables and add to the PYTHONPATH](https://stackoverflow.com/questions/3701646/how-to-add-to-the-pythonpath-in-windows-so-it-finds-my-modules-packages).
 Add the directories `C:\Users\MyUser\open_spiel\open_spiel\out\build\x64-Debug`
-and `C:\Users\MyUser\open_spiel\open_spiel\out\build\x64-Debug` to PYTHONPATH.
+and `C:\Users\MyUser\open_spiel\open_spiel\out\build\x64-Debug\python` to PYTHONPATH.
+If your PYTHONPATH does not exist, then create a new environment variable for it.
 To check that python is working, you can run the example in
 `open_spiel\python\examples`.
 

--- a/open_spiel/games/morpion_solitaire.cc
+++ b/open_spiel/games/morpion_solitaire.cc
@@ -62,36 +62,35 @@ Line::Line(Action action) {
   int base;
   Point point1;
   Point point2;
-  switch (action) {
+  if (action >= 0 && action <= 129) {
     // [0, 1]
-    case 0 ... 129:
-      row = action / 10;
-      point1 = Point(row, action - row * 10);
-      point2 = Point(row, (action - row * 10) + 3);
-      break;
+    row = action / 10;
+    point1 = Point(row, action - row * 10);
+    point2 = Point(row, (action - row * 10) + 3);
+  }
+  else if (action >= 130 && action <= 259) {
     // [1, 0]
-    case 130 ... 259:
-      base = action - 130;
-      row = (base) / 13;
-      point1 = Point(row, base - row * 13);
-      point2 = Point(row + 3, (base - row * 13));
-      break;
+    base = action - 130;
+    row = (base) / 13;
+    point1 = Point(row, base - row * 13);
+    point2 = Point(row + 3, (base - row * 13));
+  }
+  else if (action >= 260 && action <= 359) {
     // [1, -1]
-    case 260 ... 359:
-      base = action - 260;
-      row = (base) / 10;
-      point1 = Point(row, base - row * 10);
-      point2 = Point(row + 3, (base - row * 10) + 3);
-      break;
+    base = action - 260;
+    row = (base) / 10;
+    point1 = Point(row, base - row * 10);
+    point2 = Point(row + 3, (base - row * 10) + 3);
+  }
+  else if (action >= 360 && action <= 459) {
     // [1, 1]
-    case 360 ... 459:
-      base = action - 360;
-      row = (base) / 10;
-      point1 = Point(row + 3, base - row * 10);
-      point2 = Point(row, (base - row * 10) + 3);
-      break;
-    default:
-      SpielFatalError("action provided does not correspond with a move");
+    base = action - 360;
+    row = (base) / 10;
+    point1 = Point(row + 3, base - row * 10);
+    point2 = Point(row, (base - row * 10) + 3);
+  }
+  else {
+    SpielFatalError("action provided does not correspond with a move");
   }
   Init(point1, point2);
 }

